src/field.rs

```diff
@@ -26,6 +26,7 @@ use crate::tag::{ContTag, ExprTag, Op1, Op2};
 /// Because confusion on this point, perhaps combined with cargo-cult copying of incorrect previous usage has led to
 /// inconsistencies and inaccuracies in the code base, please prefer the named Scalar forms when correspondence to a
 /// named `LanguageField` is important.
+#[derive(Serialize, Deserialize, PartialEq)]
```

Comment:
This looks like a good place to experiement with the new macros generating roundtrip tests from #515